cicd/pipelines/using-pipelines-as-code.adoc

```diff
@@ -0,0 +1,58 @@
+:_content-type: ASSEMBLY
+[id="using-pipelines-as-code"]
+= Using Pipelines as Code
```

Comment:
```suggestion
= Using pipelines as code
```
Is PaC _really_ a proper noun? If so, keep it capitalized. Otherwise, if not, use sentence case and update elsewhere, as shown.

Reply:
Proper noun.